src/main/java/chess/dao/GameDaoImpl.java

```diff
@@ -0,0 +1,102 @@
+package chess.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import chess.Game;
+
+public class GameDaoImpl implements GameDao {
+
+    @Override
+    public void save(Game game) {
+        Connection connection = JdbcConnection.getConnection();
+        deleteById(game.getId());
+        String sql = "insert into game (id, id_white_player, id_black_player, turn) values (?, ?, ?, ?)";
+        try {
+            PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setInt(1, game.getId());
+            statement.setString(2, game.getIdWhitePlayer());
+            statement.setString(3, game.getIdBlackPlayer());
+            statement.setString(4, game.getTurn().toString());
+            statement.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void deleteById(int id) {
+        Connection connection = JdbcConnection.getConnection();
+        new BoardDaoImpl().deleteById(id);
```

Comment:
Dao는 데이터액세스에 집중해주세요. 이 부분은 서비스에서 조합해서 사용하는게 좋을 것 같습니다.
사용자 입장에서 GameDao로 Game을 삭제하는데, Board가 삭제되는지는 파악이 어려울 것 같네요

Reply:
```suggestion
        new BoardDaoImpl().deleteByGameId(id);
```
또 이런식으로 짓는게 좋지않을까요? 혹시 둘의 id가 같은가요?